joomla/joomla-cms
# added show/hide introtext parameter to mod_articles_news

#### Summary of Changes

Added a show_introtext parameter to the newsflash module
#### Testing Instructions

Before applying the patch publish a Newsflashmodule 
1) After applying the patch look if the introtext is still shown as default
2) Set Introtext to Hide -> Introtext should be hidden
4) Set Introtext to Show -> Introtext should be shown
#### Documentation Changes Required

![image](https://cloud.githubusercontent.com/assets/828371/17603084/d768bf6c-600f-11e6-9668-69e6f26c1df7.png)
